test/pkg/test/framework.go

```diff
@@ -471,3 +481,10 @@ func (env *Framework) DeleteAllFrameworkTracedTargetGroups(ctx aws.Context) {
 	}
 	env.TestCasesCreatedServiceNames = make(map[string]bool)
 }
+
+func (env *Framework) GetVpcLatticeServiceDns(httpRouteName string, httpRouteNamespace string) string {
+	httproute := v1beta1.HTTPRoute{}
+	env.Get(env.ctx, types.NamespacedName{Name: httpRouteName, Namespace: httpRouteNamespace}, &httproute)
+	vpcLatticeServiceDns := httproute.Annotations["application-networking.k8s.aws/lattice-assigned-domain-name"]
```

Comment:
shall we have a constant defined for this annotation?

Reply:
Ok, will change